Missing vscode theme type

Got error:
```
error TS2307: Cannot find module '@uiw/react-json-view/vscode' or its corresponding type declarations.

import { vscodeTheme } from '@uiw/react-json-view/vscode';
```

When running `tsc` from my project.

--- a/core/package.json
+++ b/core/package.json
@@ -97,6 +97,7 @@
     "nord.d.ts",
     "triangle-arrow.d.ts",
     "triangle-solid-arrow.d.ts",
+    "vscode.d.ts",
     "cjs",
     "esm"
   ],

